src/api/transaction/transactionUtils.ts

```diff
@@ -40,21 +41,24 @@ export function makeLabeledAction(
 export async function readTransactionResult(files: TransactionAsyncFile[]) {
   const transaction = files.find(x => x.name === 'transaction');
   const problems = files.find(x => x.name === 'problems');
-  const metadata = files.find(x => x.name === 'metadata');
+  const metadataProto = files.find(x => x.name === 'metadata.proto');
 
   if (!transaction) {
     throw new Error('transaction part not found');
   }
 
-  if (!metadata) {
-    throw new Error('metadata part not found');
-  }
+  // TODO uncomment and make TransactionAsyncResult.metadata required
+  // if (!metadataProto) {
+  //   throw new Error('metadata proto part not found');
+  // }
```

Comment:
Is this a TODO for this PR?

Reply:
No, it'll be removed later. Just don't want it to throw errors on old engines.